src/Cards/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from "react";
+import { CardsContainer, Score } from "./styled";
+
+import CurrentCard from "./CurrentCard";
+import Finish from "./Finish";
+import { triviaQuestions } from "./triviaQuestions";
+
+const total_questions = triviaQuestions.length;
+const currentRoundQuestions = [];
+
+const Cards = () => {
+  const [score, setScore] = useState(0);
+  const [currentQuestion, setCurrentQuestion] = useState(undefined);
```

Comment:
You should be able to set the initial state of the current question using the example I wrote below
>I know you're calling getNextQuestion in order to set the first currentQuestion, but you can simply declare the initial state of currentQuestion
const [currentQuestion, setCurrentQuestion] = useState(triviaQuestions[currentRoundQuestions.pop()]);